common/lib/xmodule/xmodule/capa_base.py

```diff
@@ -775,6 +780,7 @@ def get_problem_html(self, encapsulate=True, submit_notification=False):
             'answer_notification_message': answer_notification_message,
             'has_saved_answers': self.has_saved_answers,
             'save_message': save_message,
+            'submit_disabled_cta': submit_disabled_ctas[0] if submit_disabled_ctas else None,
```

Comment:
Isn't that going to give you a `NameError` if `submit_disabled_ctas` never got defined?

Reply:
It's defined on both branches of an if clause.

Reply:
Ah of course, just missed the first None assignment.  :)